index.html

```diff
@@ -1,29 +1,163 @@
+                <h2 class="about-Text">About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <h2 class="connect">LET'S CONNECT</h2>
+                <ul>
+                    <li><img class="section1-Icons" src="images/Frame1.svg" alt="social-Media-Icons"></li>
+                </ul>
+                <button class="btn-Project2" type="submit">See Resume</button>
+                <p class="tools">Languages <img src="images/downicon.svg" alt="tools"> </p>
+                <ul>
+                    <li class="list-Class"><img src="images/Ellipsej.svg" alt=""><div class="ellipse-J">JavaScript</div></li>
+                    <li class="list-Class"><img src="images/Ellipseh.svg" alt=""><div class="ellipse-H">HTML</div></li>
+                    <li class="list-Class"><img src="images/Ellipsec.svg" alt=""><div class="ellipse-C">CSS</div></li>
+                </ul>
+                <ul>
+                <li><p class="tools">Frameworks <img src="images/Unionright.svg" alt="tools"></p></li>
+                <li><p><img class="seperator" src="images/SeparatorBottom.svg" alt="seperator"></p></li>
+                <li><p class="tools">Skills <img src="images/Unionright.svg" alt="tools"></p></li>
+                <li><p><img class="seperator" src="images/SeparatorBottom.svg" alt="seperator"></p></li>
+            </ul>
+            <br>
+                <br>
+                <br>
+                <br>
+                <br>
+                <br>
+                <br>
+            </section>
+            <!-- <section class="contact-Me">
+                <div class="img-Background2"> 
+                    <div  class="form-Details"> 
+                <h3>Contact Me</h3>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that
+                  needs coding. I’d love to help with it</p>
+                <form action="#" name="contact_form">
+                  <label for="name"></label>
+                  <input name="first_name" type="text" required placeholder="Enter your Name" />
+                  <br>
+                  <label for="email"></label>
+                  <input name="email" type="email" required placeholder="example@gmail" />
+                  <br>
+                  <label for="message"></label><br>
+                  <textarea name="message" cols="30" rows="10" placeholder="Write your message here ..." required> </textarea>
+                  <div class="center">
+                    <input class="btn-project" type="submit" value="Get in Touch">
+                    <p><img src="images/Shape.svg" alt=""></p>
+                  </div>
+                </form>
+            </div>
+                </div>
+            </section> -->
```

Comment:
- [ ] [REQUIRED] Please kindly remove these unused (commented) codes. Having the codebase clean and solid will surely benefit you in your further milestones. In addition to that, this is against [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). So please take a good care of it :relaxed:

Reply:
completed